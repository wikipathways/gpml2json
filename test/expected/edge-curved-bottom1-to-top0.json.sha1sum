783ecde356b66ad674c695859bebd80eadbab352 ?-
