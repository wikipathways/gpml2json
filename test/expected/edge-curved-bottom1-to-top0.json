{"pathway":{"@context":["https://cdn.rawgit.com/wikipathways/WpVocabularies/7a46a05/contexts/pvjs.jsonld"],"backgroundColor":"white","borderWidth":0,"color":"black","contains":["d1b0f","dfb9b","d9b36"],"drawAs":"rect","gpmlElementName":"Pathway","height":170.8158419817691,"kaavioType":"Group","name":"test case","padding":5,"type":["Pathway"],"width":285.92858116238904,"x":0,"y":0,"zIndex":0,"fontSize":12,"fontWeight":"bold","textAlign":"left","verticalAlign":"top","dataSourceVersion":"20130813","standardName":"test case","displayName":"test case","textContent":"Name: test case","isSimilarTo":"http://wikipathways.org/index.php/Special:SearchPathways?query=test%20case&doSearch=1","id":"hash://sha256/9d4c94169484fdb59bf63f2492eaa82ebe7def28fdfb0ae01fdbc7c4bac9dbf3"},"entityMap":{"dfb9b":{"id":"dfb9b","textContent":"GeneProduct","wpType":"GeneProduct","x":175.92858116238904,"y":120.8158419817691,"fontSize":10,"height":20,"verticalAlign":"middle","width":80,"zIndex":32984,"textAlign":"center","color":"#000000","backgroundColor":"#ffffff","textDecoration":"none","fontFamily":"Arial","fontStyle":"normal","fontWeight":"normal","borderWidth":1,"drawAs":"rect","lineHeight":1.1,"overflow":"hidden","textOverflow":"clip","whiteSpace":"pre","padding":8,"type":["DataNode","SingleFreeNode","GeneProduct"],"kaavioType":"SingleFreeNode","gpmlElementName":"DataNode"},"d9b36":{"id":"d9b36","textContent":"0012","wpType":"GeneProduct","x":21.974169074530103,"y":51.14116450790639,"fontSize":10,"height":20,"verticalAlign":"middle","width":80,"zIndex":32985,"textAlign":"center","color":"#000000","backgroundColor":"#ffffff","textDecoration":"none","fontFamily":"Arial","fontStyle":"normal","fontWeight":"normal","borderWidth":1,"drawAs":"rect","lineHeight":1.1,"overflow":"hidden","textOverflow":"clip","whiteSpace":"pre","padding":8,"type":["DataNode","SingleFreeNode","GeneProduct"],"kaavioType":"SingleFreeNode","gpmlElementName":"DataNode"},"d1b0f":{"id":"d1b0f","drawAs":"CurvedLine","borderWidth":1,"zIndex":32983,"points":[{"isAttachedTo":"d9b36","attachmentDisplay":{"position":[0.5,1],"offset":[0,0]},"x":61.9741690745301,"y":71.14116450790638,"orientation":[0,1]},{"x":128.95137511845957,"y":91.14116450790638},{"isAttachedTo":"dfb9b","attachmentDisplay":{"position":[0.25,0],"offset":[0,0]},"x":195.92858116238904,"y":120.8158419817691,"orientation":[0,1]}],"color":"#000000","type":["Interaction","Edge"],"kaavioType":"Edge","gpmlElementName":"Interaction","markerStart":"TBar","markerEnd":"Arrow","isAttachedTo":["d9b36","dfb9b"]}}}
